packages/web/modals/validator-squad.tsx

```diff
@@ -41,26 +46,36 @@ interface ValidatorSquadContentProps {
   isOpen: boolean;
 }
 
+function normalizeUrl(url: string): string {
+  // Remove "https://", "http://", "www.", and trailing slashes
+  url = url.replace(/^https?:\/\//, "");
+  url = url.replace(/^www\./, "");
+  url = url.replace(/\/$/, "");
+  return url;
+}
+
+function truncateString(str: string): string {
+  if (str.length <= 30) return str;
+
+  const halfLength = Math.floor((30 - 3) / 2);
+  const firstHalf = str.slice(0, halfLength);
+  const secondHalf = str.slice(-halfLength);
+  return `${firstHalf}...${secondHalf}`;
+}
+
```

Comment:
Note for these kinds of functions, we have this folder of utils. Please try to categorize them there so they're more easily discoverable